common/resources.yaml

```diff
@@ -0,0 +1,67 @@
+groups:
+ - data_type_def: ExternalResources
+   data_type_inc: Container
+   doc: A pair of tables for tracking external resource references in a file
+   datasets:
+    - data_type_inc: Data
+      name: keys
+      doc: A table for identifying which objects in a file contain
```

Comment:
"A table for identifying which objects in a file contain"...

is there more that got left off here?